src/CodeGenerator/Generator/ApiGenerator.php

```diff
@@ -679,7 +810,7 @@ private function resultClassAddProperties(string $baseNamespace, bool $root, ?ar
                     $this->doGenerateResultClass($baseNamespace, $listItemShapeName);
                     $returnType = $this->safeClassName($listItemShapeName);
                 } else {
-                    $returnType = $type . '[]';
+                    $returnType = $type;
```

Comment:
Are you sure? 

Isn't it an array of `$type`?

Reply:
I applied your sugestion, a `\Traversable<int, {$returnType}>`. Maybe the variable needs a better name

Reply:
Oh. That is why this line was changed. 

Good good. It is just me that was confused.